frontend/src/graph/ui/Chart.vue

```diff
@@ -1,73 +1,69 @@
 <template>
   <div class="visual-dl-graph-charts">
-    <svg class="visual-dl-page-left">
+    <svg
+      class="visual-dl-page-left"
+      id="graphSvg">
```

Comment:
i think Vue style people normally use like ref="graphSvg"  and reference as ```let svg = this.$refs.graphSvg``` instead of ```let svg = document.getElementById('graphSvg');```

Reply:
Got it. Thanks!

Reply:
Done